app/controllers/conferences_controller.rb

```diff
@@ -6,6 +6,10 @@ class ConferencesController < ApplicationController
   def index
     @current = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc)
     @antiquated = @conferences - @current
+
+    redirect_to @current.first if @current.count == 1 &&
```

Comment:
Based on what we agreed on #1774 

> If there is only 1 conference, and that conference has a splashpage, redirect to conferences#show from conferences#index

this should read `Conference.count == 1`